src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    public static final int COLORS_COUNT = new Random().nextInt(Colors.values().length);
```

Comment:
create just Random as a class-level field
and randomize a color in method

now you will always get the same color